crates/runtime/benches/bench_s3/mod.rs

```diff
@@ -151,3 +260,215 @@ fn get_test_queries() -> Vec<(&'static str, &'static str)> {
+        ("tpcds_q59", include_str!("../queries/tpcds/q59.sql")),
+        ("tpcds_q60", include_str!("../queries/tpcds/q60.sql")),
+        ("tpcds_q61", include_str!("../queries/tpcds/q61.sql")),
+        ("tpcds_q62", include_str!("../queries/tpcds/q62.sql")),
+        ("tpcds_q63", include_str!("../queries/tpcds/q63.sql")),
+        ("tpcds_q64", include_str!("../queries/tpcds/q64.sql")),
+        ("tpcds_q65", include_str!("../queries/tpcds/q65.sql")),
+        ("tpcds_q66", include_str!("../queries/tpcds/q66.sql")),
+        ("tpcds_q67", include_str!("../queries/tpcds/q67.sql")),
+        ("tpcds_q68", include_str!("../queries/tpcds/q68.sql")),
+        ("tpcds_q69", include_str!("../queries/tpcds/q69.sql")),
+        ("tpcds_q70", include_str!("../queries/tpcds/q70.sql")),
+        ("tpcds_q71", include_str!("../queries/tpcds/q71.sql")),
+        ("tpcds_q72", include_str!("../queries/tpcds/q72.sql")),
+        ("tpcds_q73", include_str!("../queries/tpcds/q73.sql")),
+        ("tpcds_q74", include_str!("../queries/tpcds/q74.sql")),
+        ("tpcds_q75", include_str!("../queries/tpcds/q75.sql")),
+        ("tpcds_q76", include_str!("../queries/tpcds/q76.sql")),
+        ("tpcds_q77", include_str!("../queries/tpcds/q77.sql")),
+        ("tpcds_q78", include_str!("../queries/tpcds/q78.sql")),
+        ("tpcds_q79", include_str!("../queries/tpcds/q79.sql")),
+        ("tpcds_q80", include_str!("../queries/tpcds/q80.sql")),
+        ("tpcds_q81", include_str!("../queries/tpcds/q81.sql")),
+        ("tpcds_q82", include_str!("../queries/tpcds/q82.sql")),
+        ("tpcds_q83", include_str!("../queries/tpcds/q83.sql")),
+        ("tpcds_q84", include_str!("../queries/tpcds/q84.sql")),
+        ("tpcds_q85", include_str!("../queries/tpcds/q85.sql")),
+        ("tpcds_q86", include_str!("../queries/tpcds/q86.sql")),
+        ("tpcds_q87", include_str!("../queries/tpcds/q87.sql")),
+        ("tpcds_q88", include_str!("../queries/tpcds/q88.sql")),
+        ("tpcds_q89", include_str!("../queries/tpcds/q89.sql")),
+        ("tpcds_q90", include_str!("../queries/tpcds/q90.sql")),
+        ("tpcds_q91", include_str!("../queries/tpcds/q91.sql")),
+        ("tpcds_q92", include_str!("../queries/tpcds/q92.sql")),
+        ("tpcds_q93", include_str!("../queries/tpcds/q93.sql")),
+        ("tpcds_q94", include_str!("../queries/tpcds/q94.sql")),
+        ("tpcds_q95", include_str!("../queries/tpcds/q95.sql")),
+        ("tpcds_q96", include_str!("../queries/tpcds/q96.sql")),
+        ("tpcds_q97", include_str!("../queries/tpcds/q97.sql")),
+        ("tpcds_q98", include_str!("../queries/tpcds/q98.sql")),
+        ("tpcds_q99", include_str!("../queries/tpcds/q99.sql")),
+    ]
+}
+
+#[allow(clippy::too_many_lines)]
+fn get_postgres_tpcds_test_queries() -> Vec<(&'static str, &'static str)> {
+    vec![
+        // Query 1, 30, 64, 81 commented out due to rewritten query's expensive plan in Postgres
```

Comment:
nice, I see the comment here now. Let's incude a shorter blurb version next to the actual `if {}`